### ISSUE TITLE
No module named 'bingads.v13.bulk.entities.goals'

### ISSUE THREAD

**Issue by klenium** [BingAds/BingAds-Python-SDK#290]
`import bingads.v13.bulk as BingAdsBulk`

If executed with package version 13.0.21, yesterday's release, merged in #289, it results as:

```
ModuleNotFoundError                       Traceback (most recent call last)
File <>, line 2
      1 import bingads
----> 2 import bingads.v13.bulk as BingAdsBulk
      3 import bingads.v13.reporting as BingAdsReporting

File /python/lib/python3.10/site-packages/bingads/v13/bulk/__init__.py:11
      9 from .upload_parameters import *
     10 from .bulk_operation import *
---> 11 from .file_reader import *
     12 from .file_writer import *
     13 from .bulk_service_manager import *

File /python/lib/python3.10/site-packages/bingads/v13/bulk/file_reader.py:2
      1 from .enums import ResultFileType
----> 2 from .entities.bulk_entity import BulkEntity
      3 from bingads.v13.internal.bulk.stream_reader import _BulkStreamReader
      4 from bingads.v13.internal.bulk.record_reader import _BulkRecordReader

File /python/lib/python3.10/site-packages/bingads/v13/bulk/entities/__init__.py:57
     55 from .bulk_brand_item import *
     56 from .bulk_brand_list import *
---> 57 from .goals import *

ModuleNotFoundError: No module named 'bingads.v13.bulk.entities.goals'
```

**Comment by xinyuwen2** [comment]
Sorry that I just missed this module in setup.py. This fix has been shipped in newest version 13.0.21.1.

**Comment by klenium** [comment]
Thanks!
